api/Makefile

```diff
@@ -11,24 +11,16 @@ RM_FILE      = rm -f
 INSTALL_DIR  = install -o root -g ${OSSEC_GROUP} -m 0750  -d
 INSTALL_EXEC = install -o root -g ${OSSEC_GROUP} -m 0750
 INSTALL_FILE = install -o root -g ${OSSEC_GROUP} -m 0640
+PYTHON_BIN   = $(PREFIX)/framework/python/bin/python3
 
 
 .PHONY: all install
 
 all: install
 
 install:
-	$(INSTALL_DIR) $(PREFIX)/api/api
-	$(INSTALL_DIR) $(PREFIX)/api/api/controllers
-	$(INSTALL_DIR) $(PREFIX)/api/api/models
-	$(INSTALL_DIR) $(PREFIX)/api/api/spec
 	$(INSTALL_DIR) $(PREFIX)/api/configuration
```

Comment:
```suggestion
	$(INSTALL_RW_DIR) $(PREFIX)/api/security
	$(INSTALL_DIR) $(PREFIX)/api/configuration
```

Reply:
Added in 2fc48a8612adcf3319996e315cd6fa801500548a